src/Sylius/Bundle/PromotionBundle/Resources/translations/messages.fr.yml

```diff
@@ -34,7 +34,20 @@ sylius:
         after: Utilisateur inscrit après cette période ?
       shipping_country_configuration:
         country: Pays
-      type: Type
+      order_loyalty_configuration:
```

Comment:
I think the translation process takes is different on Sylius, keep an eye on #3317 & https://github.com/Sylius/Sylius/pull/3316#issuecomment-142021207